app-api/src/main/java/org/openmrs/eip/app/management/entity/ReceiverSyncStatus.java

```diff
@@ -2,19 +2,21 @@
 
 import java.util.Date;
 
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 @Entity
 @Table(name = "receiver_sync_status")
-public class ReceiverSyncStatus extends AbstractEntity {
-	
+public class ReceiverSyncStatus extends AbstractEntity {	
 	public static final long serialVersionUID = 1;
 	
-	//TODO This should be mapped as a @OneToOne association when we build a UI
-	@Column(name = "site_info_id", nullable = false, updatable = false, unique = true)
```

Comment:
Please preserve all these existing constraints on the column, don't get rid of them

Reply:
But the column site_info_id has been removed, since now I'm using @OneToOne association

Reply:
I mean the constrains need to stay

Reply:
i.e on the property in these JPA annotations

Reply:
Like this?
`	@OneToOne(cascade = CascadeType.ALL, optional = false)
	@JoinColumn(name = "site_info_id", nullable = false, updatable = false, unique = true)
	private SiteInfo siteInfo;`

Reply:
Yes, you can exclude the name field though, it's redundant because hibernate will default to the the primary key column site_info_id still

Reply:
Yes